docs/getting-started/key-components/workflow.md

```diff
@@ -55,4 +55,40 @@ If your workflow execution fails, you can retry the execution. To retry a failed
     * **Retry with currently saved workflow**: Once you make changes to your workflow, you can select this option to execute the workflow with the previous execution data.
     * **Retry with original workflow**: If you want to retry the execution without making changes to your workflow, you can select this option to retry the execution with the previous execution data.
 
-You can also use the [Error Trigger node](../../nodes/nodes-library/core-nodes/ErrorTrigger), which triggers a workflow when another workflow has an error. Once a workflow fails, this node gets details about the failed workflow and the errors.
\ No newline at end of file
+You can also use the [Error Trigger node](../../nodes/nodes-library/core-nodes/ErrorTrigger), which triggers a workflow when another workflow has an error. Once a workflow fails, this node gets details about the failed workflow and the errors.
+
+## Workflow templates
+
+When creating a new workflow, you can choose whether to start with an empty workflow, or use an existing template.
+
+Templates provide:
+
+* A way to get started quickly: we might already have a template that does what you need.
+* Examples of what you can build, and best practices for creating your own workflows.
+
+### Use a workflow template
+
+1. In the sidebar, click **Templates**.
+2. Browse or search the workflow templates list.
+3. Click a workflow to view more information. n8n opens the workflow details page.
+4. On the workflow details page, click **Use this workflow**. n8n opens the workflow.
+5. Click **Save** to add the workflow to your workflows.
+
+::: tip 
+Workflow templates are available on all flavours of n8n. If you can't access workflow templates in n8n, you may be on an old version, or using a self-hosted or embeded version of n8n with templates disabled.
```

Comment:
```suggestion
Workflow templates are available on all flavors of n8n. If you can't access workflow templates in n8n, you may be on an old version, or using a self-hosted or embedded version of n8n with templates disabled.
```

It would be more clear to mention the first version on which templates are available, for example: "you may be on an old version (< 0.xxx.x)".

Reply:
Will add the version once we know it :-)